summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -35,6 +41,10 @@ showGhcVer = \case
     Ghc844  -> "8.4.4"
     Ghc863  -> "8.6.3"
 
+-- | These are old GHC versions that are not working with default GHC versions when using Stack.
+oldGhcs :: [GhcVer]
+oldGhcs = [Ghc7103, Ghc801, Ghc802]
```

Comment:
Can this be written as `[minBound .. Ghc802]`? I assume that once we bump up cabal-version, we only need to bump up upper bound in this list. I guess that the problem with cabal doesn't work for all GHC versions in sequence up to some version..